site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -92,6 +92,7 @@ The `spec` field is the actual `FleetAutoscaler` specification and it is compose
       - `namespace` is the kubernetes namespace where webhook is deployed. Optional
                       If not specified, the "default" would be used
       - `path` is an optional URL path which will be sent in any request to this service. (i. e. /scale)
+      - `port` is optional, it is the port for the service which is hosting the webhook. The default is 8000 for backward compatibility. If given, it should be a valid port number (1-65535, inclusive).
```

Comment:
Basically this way:
```suggestion
{{% feature publishVersion="1.9.0" %}}
      - `port` is optional, it is the port for the service which is hosting the webhook. The default is 8000 for backward compatibility. If given, it should be a valid port number (1-65535, inclusive).
{{% /feature %}}
```

Reply:
Looks like this might be the last thing before we can merge.

(Confirmed with @thisisnotapril that we can manually approve CLA) @andrewgrundy, you good to make the change?